ledger/participant-state/kvutils/src/main/scala/com/daml/ledger/participant/state/kvutils/KeyValueConsumption.scala

```diff
@@ -469,5 +469,5 @@ object KeyValueConsumption {
       .fold(err => throw Err.DecodeError("Party", s"Cannot parse '$s': $err"), identity)
 
   private def correlationId(submitterInfo: DamlSubmitterInfo): String =
-    submitterInfo.getCommandId
+    submitterInfo.getSubmissionId
```

Comment:
CC @miklos-da We've decided to use the submission id as the error code correlation id for the time being